path/is_absolute.ts

```diff
@@ -7,7 +7,22 @@ import { isAbsolute as windowsIsAbsolute } from "./windows/is_absolute.ts";
 
 /**
  * Verifies whether provided path is absolute
+ *
  * @param path to be verified as absolute
+ * @returns `true` if path is absolute, `false` otherwise
+ *
+ * @example Usage
+ * ```ts
+ * import { isAbsolute } from "@std/path/is-absolute";
+ *
+ * // posix
+ * isAbsolute("/home/foo"); // true
```

Comment:
Ideally we could use assertions in these examples, but I couldn't figure out a nice way to make it work due to the os-dependent behavior.

I suppose we could do something like
```ts
if (Deno.build.os === "windows") {
  // windows specific examples
} else {
  // posix
}
```

but I'm not sure if it's worth the clutter

Reply:
I think the readability matters more here. I'd suggest adding `no-eval` directive when it's difficult to make it work in cross-platform way